templates/prometheus.go

```diff
@@ -39,6 +42,23 @@ var resourceSelector = metav1.LabelSelector{
 	},
 }
 
+var metrics = []string{
+	"noobaa_num_buckets_claims",
+	"noobaa_num_unhealthy_bucket_claims",
+	"job:noobaa_total_unhealthy_namespace_buckets",
+	"job:noobaa_namespace_bucket_count",
+	"noobaa_accounts_num",
+	"noobaa_total_usage",
+	"noobaa_num_unhealthy_namespace_resources",
+	"nooBaa_num_namespace_resources",
+	"noobaa_namespace_resource_status",
+	"noobaa_namespace_bucket_status",
+	"noobaa_system_health_status",
+	"BucketPolicyErrorState",
+	"CacheBucketErrorState",
+	"DataSourceErrorState",
+}
```

Comment:
FYI This will need to be updated based on the state of https://github.com/red-hat-storage/mcg-osd-deployer/pull/67 when its merged.

Reply:
Metrics names updated according #67